spaceship/docs/DeveloperPortal.md

```diff
@@ -230,8 +230,20 @@ Spaceship.provisioning_profile.all.find_all { |p| !p.valid? || !p.certificate_va
 ## Devices
 
 ```ruby
+# Get all enabled devices
 all_devices = Spaceship.device.all
 
+# Disable first device
+all_devices.first.disable!
+
+# Find disabled device and enable it
+Spaceship.device.find_by_udid("44ee59893cb...").enable!
+
+# Get list of all devices, including disabled ones, and filter the result to only include disabled devices
+disabled_devices = Spaceship.device.all(includeDisabled: true).select do |device|
+  !device.enabled?
+end
```

Comment:
Could you add `disabled?` as a convenience method as well? I think that would allow this to become the very nice:

```ruby
disabled_devices = Spaceship.device.all(includeDisabled: true).select(&:disabled?)
```